StateCodelab/start/src/main/java/com/codelabs/state/todo/Data.kt

```diff
@@ -23,7 +23,7 @@ import androidx.compose.material.icons.filled.Event
 import androidx.compose.material.icons.filled.PrivacyTip
 import androidx.compose.material.icons.filled.RestoreFromTrash
 import androidx.compose.ui.graphics.vector.VectorAsset
-import java.util.UUID
+import java.util.*
```

Comment:
Single import

Reply:
Any way to got AS to stop making this edit? :)